openshift-knative-operator/pkg/common/env.go

```diff
@@ -0,0 +1,26 @@
+package common
+
+import (
+	mf "github.com/manifestival/manifestival"
+	corev1 "k8s.io/api/core/v1"
+	"knative.dev/operator/pkg/apis/operator/base"
+)
+
+// ConfigureEnvValueIfUnset sets env values via arguments if users haven't set in the CR.
+func ConfigureEnvValueIfUnset(s *base.CommonSpec, deployment, container, key, value string) mf.Transformer {
```

Comment:
If we add this we need to adjust also `configIfUnsetAndCheckIfShouldInject` used elsewhere I suppose.